docs/source/monitor_rocoto.rst

```diff
@@ -53,9 +70,10 @@ Usage
 
    ./rocoto_viewer.py -d /path/to/workflow/database/file -w /path/to/workflow/xml/file
 
-Note 1: Terminal/window must be wide enough to display all experiment information columns, viewer will complain if not.
+.. note::
+   Note 1: Terminal/window must be wide enough to display all experiment information columns, viewer will complain if not.
 
-Note 2: The viewer requires the full path to the database and xml files if you are not in your EXPDIR when you invoke it.
+   Note 2: The viewer requires the full path to the database and xml files if you are not in your EXPDIR when you invoke it.
```

Comment:
Note 3: Only `TERM=xterm` is supported. You may wish to create a shell function to switch automatically if you are in a different terminal:

Bash example
```
function rv {
	oldterm=${TERM};
	export TERM='xterm';
	${PATH_TO_VIEWER}/rocoto_viewer.py $@;
	export TERM=${oldterm};
}
```

Reply:
Added @ 6be21829